goinp/goinp_test.go

```diff
@@ -429,51 +428,30 @@ func TestSelectFromStringsFromReader(t *testing.T) {
 	}
 }
 
-func testAskOptionWrapper(title string, defaultValue string, optional bool, stdin string, options ...string) (string, string, error) {
-	in, err := ioutil.TempFile("/tmp", "gotest")
-	if err != nil {
-		return "", "", err
-	}
-	defer in.Close()
-
-	out, err := ioutil.TempFile("/tmp", "gotest")
-	if err != nil {
-		return "", "", err
-	}
-	defer in.Close()
-
-	// test single input - not optional
-	if _, err := io.WriteString(in, stdin); err != nil {
-		return "", "", err
-	}
-
-	if _, err = in.Seek(0, os.SEEK_SET); err != nil {
-		return "", "", err
-	}
+type testStdin struct {
+	Stdin *bytes.Buffer
+}
 
-	os.Stdin = in
-	os.Stdout = out
+func (testStdin) Fd() uintptr {
+	return uintptr(syscall.Stdin)
```

Comment:
why do you do this?

Reply:
I wanted to mention without any comment that it is zero because that is a valid tty device ID